tracer/src/Datadog.Trace/Iast/Aspects/System.Text.Json/JsonDocumentAspects.cs

```diff
@@ -0,0 +1,126 @@
+// <copyright file="JsonDocumentAspects.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+#if NETCOREAPP3_0_OR_GREATER
```

Comment:
We're going to be building for [.NET Core 2.1 soon](https://github.com/DataDog/dd-trace-dotnet/pull/5260) too (instead of `netstandard2.0`) - if these APIs are going to be available there, You could use `#if NETCOREAPP` instead, it'll do the same thing now, and it'll do the correct thing after that PR too... But if it's going to break, then we should look at changing it later instead!

Reply:
Changing to `#if NETCOREAPP` is unfortunately not possible here because the [JsonDocument object](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsondocument.parse?view=net-8.0&viewFallbackFrom=netcore-2.1#system-text-json-jsondocument-parse(system-string-system-text-json-jsondocumentoptions)) only exists since netcoreapp 3.0 and will fail inside the AspNetCore2 sample